.vscode/launch.json

```diff
@@ -4,6 +4,23 @@
       // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
       "version": "0.2.0",
       "configurations": [
+            {
+                  "name": "(Windows) debug",
+                  "type": "cppvsdbg",
+                  "request": "launch",
+                  "program": "${workspaceFolder}/msvc.install_x64/bin/MuseScore3.exe",
+                  "args": ["-d"],
+                  "stopAtEntry": false,
+                  "cwd": "${workspaceFolder}",
+                  "environment": [],
+                  "externalConsole": true
+            },
+            {
+                  "name": "(Windows) Attach",
+                  "type": "cppvsdbg",
+                  "request": "attach",
+                  "processId": "${workspaceFolder}/msvc.install_x64/bin/MuseScore3.exe"
+            },
             {
                   "name": "gdb debug from debug build",
```

Comment:
Since you've added a `Windows` qualifier to the above names, you could add `Linux` to this one?